### PR TITLE
Add a test to check online unique index duplicates during every stage

### PR DESCRIPTION
**Pull request by vitalif**
### Changelog category

* Not for changelog (changelog entry is not required)

### Description for reviewers

### CODE REVIEW

File: ydb/core/kqp/ut/indexes/kqp_indexes_ut.cpp [hunk 1]
```diff
@@ -6151,6 +6153,147 @@ R"([[#;#;["Primary1"];[41u]];[["Secondary2"];[2u];["Primary2"];[42u]];[["Seconda
...
+    }
+
+    Y_UNIT_TEST(OnlineDuplicatesStage3) {
+        DoTestOnlineDuplicates(false, 1, [&](const TAutoPtr<IEventHandle>& ev) {
+            return ev->Type == static_cast<ui32>(NKikimr::TEvDataShard::EvValidateUniqueIndexRequest);
+        });
+    }
+
+    Y_UNIT_TEST(OnlineDuplicatesStage4) {
+        DoTestOnlineDuplicates(false, 2, [&](const TAutoPtr<IEventHandle>& ev) {
```
[the remark this answers is not in the document]
Comment (vitalif) [reply ydb/core/kqp/ut/indexes/kqp_indexes_ut.cpp]: \-

2 потому что 1 индексный и 1 основной таблицы

**Comment by vitalif** [comment]
Добавил пояснения, что означает каждая фаза

**Review by azevaykin — APPROVED** [review]
